src/host/toolsHost.ts

```diff
@@ -16,6 +16,8 @@ export default class ToolsHost {
     private resourceLoader: Readonly<ToolsResourceLoader> | undefined;
     private getHostCallbacksNextId: number = 0;
     private getHostCallbacks: Map<number, (preferences: object) => void> = new Map();
+    // We need to add a dummy property to get around build errors for sendToVscodeOutput.
+    InspectorFrontendHostInstance: any;

```

Comment:
@vidorteg 

This does the same thing as the original `[x: string]: any;` which is to make it so this function doesn't throw an error since `InspectorFrontendHost.InspectorFrontendHostInstance` isn't defined:
```
export function sendToVscodeOutput(message: string) {
    InspectorFrontendHost.InspectorFrontendHostInstance.sendToVscodeOutput(message);
}
```

The main issue is that if we want to reference InspectorFrontendHost in `root.js` like in `getVscodeSettings()`, we use `InspectorFrontendHost`.  However, in any file outside of `root.js` like for `sendToVscodeOutput`, we need to call `InspectorFrontendHost.InspectorFrontendHostInstance`.

We're kind of using these functions unconventionally since we declare them in `simpleView.ts` but we never actually use them directly and only use it to inject the function in our source code.  So even though it doesn't need to make sense in the context of our extension code, it is giving us an error.

I'll wait to check this change in until you see this.  Let me know if there is a better solution than creating this dummy property.